services.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+- name: diamond.service
+- name: image-cleaner.service
+  version: 1.0.0
+- name: neo4j-hot-backup@.service
+  version: 1.1-go
+  count: 1
+  desiredState: loaded
+- name: neo4j-hot-backup@.timer
+  count: 1
+- name: splunk-forwarder.service
+- name: tunnel-registrator.service
+  version: v1.0.3
+- name: neo4j-sidekick@.service
+  count: 3
+- name: neo4j@.service
+  version: 3.1.0-rc1
```

Comment:
rc, release candidate? We should have a stable version that is available now https://neo4j.com/download/other-releases/